pkg/errors/errors.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I took this code from the k8s/ingress-nginx repo and thus, the license.

Reply:
Checking w/ https://docs.opensource.microsoft.com/oss-licenses-by-type.html#permissive-oss-licenses - Apache is in Permissive OSS licenses and allowed.